src/bootman/kernel.c

```diff
@@ -608,6 +609,40 @@ static bool boot_manager_remove_legacy_uefi_kernel(const BootManager *manager, c
         return ret;
 }
 
+bool _copy_extra_initrds(const char *initrd_base, const char *target_dir) {
+        char *initrd_source = NULL;
+        autofree(char) *initrd_glob = string_printf("%s.*", initrd_base);
+        autofree(char) *initrd_target = NULL;
+        glob_t files;
+
+        int res = glob(initrd_glob, 0, NULL, &files);
+        if (res == GLOB_NOMATCH) {
```

Comment:
Instead of using two conditionals for the same result, why not instead..

```c

    bool ret = false;

    if (cond..) {
            goto error;
    }
...

    ret = true;
error:

    globfree(&files);
    return ret;

Reply:
I refactored it into a separate function and less if statements. Let me know what you think!